livechat/agent/rtm/client.py

```diff
@@ -967,4 +970,10 @@ def list_agents_for_transfer(self,
 
 
 class AgentRTM33(AgentRTMInterface):
-    ''' AgentRTM version 3.3 class. '''
+    ''' Agent RTM version 3.3 class. '''
+    pass
```

Comment:
pass is not necessary (https://www.python.org/dev/peps/pep-0257/)

Reply:
👍